### PR TITLE
Multiple wallets

### PR DESCRIPTION
**Pull request by romaroid**

### CODE REVIEW

File: src/js/controllers/ReceiveControllers.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 angular.module('blocktrail.wallet')
-    .controller('ReceiveCtrl', function($scope, $rootScope, CONFIG, Wallet, CurrencyConverter, Currencies, $q, $timeout, $translate, trackingService) {
+    .controller('ReceiveCtrl', function($scope, $rootScope, CONFIG, activeWallet, settingsService, CurrencyConverter,
+                                        Currencies, $q, $timeout, $translate, trackingService) {
+        // TODO Review
```
Comment (rubensayshi) [inline src/js/controllers/ReceiveControllers.js]: `@TODO`

File: src/js/controllers/ReceiveControllers.js [hunk 2]
```diff
@@ -103,6 +108,7 @@ angular.module('blocktrail.wallet')
         //generate the first address
         $scope.newAddress();
 
+        // TODO Create a subscription on change transaction list
```
Comment (rubensayshi) [inline src/js/controllers/ReceiveControllers.js]: `@TODO`

File: src/js/controllers/SendControllers.js [hunk 3]
```diff
@@ -123,110 +124,106 @@ angular.module('blocktrail.wallet')
...
-                    return $q.all([
-                        wallet.coinSelection(localPay, false, $scope.useZeroConf, blocktrailSDK.Wallet.FEE_STRATEGY_LOW_PRIORITY)
-                            .spread(function (utxos, fee, change, res) {
-                                $log.debug('lowPriority fee: ' + fee);
+            // either use the real destination address or otherwise use a fake address
+            if ($scope.sendInput.recipientAddress) {
+                localPay[$scope.sendInput.recipientAddress] = amount;
+            } else {
+                var fakeP2SHScript = bitcoinJS.scripts.scriptHashOutput(new blocktrailSDK.Buffer("0000000000000000000000000000000000000000", 'hex'));
+                var fakeAddress = bitcoinJS.Address.fromOutputScript(fakeP2SHScript, activeWallet._sdkWallet.sdk.network);
```
Comment (rubensayshi) [inline src/js/controllers/SendControllers.js]: note to self; test

**Comment by romaroid** [comment]
merged

**Pull request closed without merging**